module/programs/helper.js

```diff
@@ -0,0 +1,427 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 03-sep-2020
+ * Description : Programs related helper functionality.
+ */
+
+// Dependencies
+
+const timeZoneDifference = process.env.TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC;
+const userService = require(PROJECT_ROOT_DIRECTORY + "/generics/services/users");
+
+/**
+ * ProgramsHelper
+ * @class
+ */
+module.exports = class ProgramsHelper {
+
+     /**
+   * Programs Document.
+   * @method
+   * @name programDocuments
+   * @param {Array} [filterQuery = "all"] - solution ids.
+   * @param {Array} [fieldsArray = "all"] - projected fields.
+   * @param {Array} [skipFields = "none"] - field not to include.
+   * @param {Number} pageNo - page no.
+   * @param {Number} pageSize - page size.
+   * @returns {Array} List of programs.
+   */
+
+  static programDocuments(
```

Comment:
@Prajwal17Tunerlabs please remove this from helper
refer this https://github.com/project-sunbird/ml-projects-service/blob/release-7.0.0/databaseQueries/programs.js